DNN Platform/Modules/MemberDirectory/Services/MemberDirectoryController.cs

```diff
@@ -159,7 +159,7 @@ private IEnumerable<UserInfo> GetUsers(int userId, int groupId, string searchTer
                     var propertyValue = GetSetting(ActiveModule.ModuleSettings, "FilterPropertyValue", String.Empty);
                     AddSearchTerm(ref propertyNames, ref propertyValues, filterValue, propertyValue);
 
-                    users = UserController.Instance.GetUsersAdvancedSearch(portalId, userId, -1, -1,
+                    users = UserController.Instance.GetUsersAdvancedSearch(portalId, PortalSettings.UserId, userId, -1,
```

Comment:
The filterUserId parameter only used when search type is "Relationship", so IMO it can keep use -1 here.